lineapy/graph_reader/artifact.py

```diff
@@ -1,5 +1,5 @@
 """
-User exposed objects through the :mod:`lineapy.apis`.
+User exposed objects through the :mod:`lineapy.api`.
 """
 from __future__ import annotations
 
```

Comment:
i think the intent here is to move it from lineapy/graph_reader/xxx to lineapy/api/xxx folder rather than renaming apis.py to artifact.py.

Reply:
yes sorry I was not clear!!!